### ISSUE TITLE
Update releases.json

### ISSUE THREAD

**Issue by stevieb9** [StrawberryPerl/strawberryperl.com#34]
I'd like to incorporate the new versions in `berrybrew`, but to do so, the `releases.json` file must be updated.

It appears as though it's a relatively trivial task to get this done, but it seems as though I'd need to have all the zip files locally (`c:/strawberry_build/output`).

Is there an easy way that I could mirror the list of releases? If so, I'll try my hand at getting the release list files updated. If not, could somebody else please do so?

**Comment by shawnlaffan** [comment]
Are you grabbing it from the website or the github repo?  

The website is still a work in progress as permissions are not yet in hand.  See discussion under https://github.com/StrawberryPerl/Perl-Dist-Strawberry/issues/114

If you are using the repo then then it should be pretty straightforward and the file just needs the 5.36.1.1 and 5.38.0.1 entries to be added, pointing to the GH release URLs under https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/tag/SP_5380_5361  

A PR would be great if you are able to do so.

**Comment by stevieb9** [comment]
Tomorrow, I'll update the files in the repo, test them, and create a PR. I'll also follow and monitor the website issue. I'll then have `berrybrew` use the repo `releases.json` file by default instead of the Strawberry site (at least for now).

**Comment by stevieb9** [comment]
I have done this, and created a [PR](https://github.com/StrawberryPerl/strawberryperl.com/pull/35).

Once this is merged, `berrybrew` will be able to see and install the new versions. I won't be able to provide the full directions until the PR is merged in, but will do so thereafter.

I have mostly modified the generation of the `releases.json` file, so I'll monitor for new releases, and create a new PR each time a new one is available (unless I could be given write access to the repo, then I could fully automate it).

**Comment by shawnlaffan** [comment]
I think this is closable?
